internal/pkg/aws/ecs/ecs.go

```diff
@@ -20,13 +21,17 @@ const (
 	shortTaskIDLength      = 8
 	shortImageDigestLength = 8
 	imageDigestPrefix      = "sha256:"
+	runTaskStartedBy       = "copilot-task"
```

Comment:
Same as above, we should mingle Copilot concepts to the `aws/` pkgs instead it should be passed down as an input from a caller to make the pkgs more generic